<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EASIViewControls</class>
 <widget class="QWidget" name="EASIViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
     </property>
     <property name="text">
      <string>Electrophysiology Simulations Pipeline</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_1">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="text">
      <string>   Step1: Load DICOM</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_2">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="text">
      <string>   Step2: Process Images</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="button_2_1">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Convert to Nifti</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="button_2_2">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Crop Images</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="button_2_3">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Resample Images</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_3">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="text">
      <string>   Step3: Segment Images</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_4">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="text">
      <string>   Step4: Create Mesh</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_5">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;</string>
     </property>
     <property name="text">
      <string>   Step5: Activation Sites</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="button_5_1">
     <property name="minimumSize">
      <size>
       <width>140</width>
 